go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/firecracker-microvm/firecracker-containerd
 
 require (
+	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/Microsoft/go-winio v0.4.11 // indirect
 	github.com/Microsoft/hcsshim v0.8.1 // indirect
+	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705 // indirect
```

Comment:
Is this being pulled in by containerd?

Reply:
```
go mod why github.com/StackExchange/wmi      
                                                                                                                                                                                                                                                                                                                           
# github.com/StackExchange/wmi
github.com/firecracker-microvm/firecracker-containerd/internal
github.com/shirou/gopsutil/process
github.com/StackExchange/wmi
```

It appears to be the psutil library needed by the new service_integ_test package. That library also supports windows, so it makes sense I guess.